nano/core_test/websocket.cpp

```diff
@@ -158,7 +159,13 @@ TEST (websocket, started_election)
 				 .work (*system.work.generate (nano::dev::genesis->hash ()))
 				 .build_shared ();
 	nano::publish publish1{ nano::dev::network_params.network, send1 };
-	auto channel1 = node1->network.udp_channels.create (node1->network.endpoint ());
+
+	// Another node instance just to be a tcp client for channel1.
+	auto other_node (std::make_shared<nano::node> (system.io_ctx, nano::get_available_port (), nano::unique_path (), system.logging, system.work));
+	other_node->start ();
+	system.nodes.push_back (other_node);
+
```

Comment:
These lines are duplicated in multiple tests. Maybe a helper function could help removing that duplication and simplifying the tests?